docs/releases/5.2.txt

```diff
@@ -413,6 +413,10 @@ Miscellaneous
   provided. Previously, they were ignored if :attr:`.UniqueConstraint.fields`
   was set without a :attr:`.UniqueConstraint.condition`.
 
+* ``migrate`` and ``runserver`` now respect the value of ``requires_system_checks``
+  if overridden. As a consequence, when using ``call_command()`` to invoke these
+  commands, system checks must be enabled with ``skip_checks=False``.
```

Comment:
`call_command("runserver")` isn't very realistic, but `call_command("migrate")` might be? This seemed borderline whether to doc or not.

Reply:
yeah I'm not sure either, this seems borderline bugfixing which we normally don't document.

Reply:
I retested, and I can't reproduce a behavior difference with `call_command()`, so I'll remove this. Sorry for the confusion.